frontend/package.json

```diff
@@ -33,6 +34,7 @@
     "react-query": "^3.39.3",
     "react-refresh": "^0.11.0",
     "react-router-dom": "^6.12.1",
+    "react-snowfall": "^1.2.1",
```

Comment:
Hahaha I fully support this.

Reply:
If possible would request you to upload images for different season. Mine aren't that high quality.